provider/ovh/ovh_test.go

```diff
@@ -93,7 +93,7 @@ func TestOvhZoneRecords(t *testing.T) {
 	zones, records, err := provider.zonesRecords(context.TODO())
 	assert.NoError(err)
 	assert.ElementsMatch(zones, []string{"example.org"})
-	assert.ElementsMatch(records, []ovhRecord{{ID: 42, Zone: "example.org", ovhRecordFields: ovhRecordFields{SubDomain: "ovh", FieldType: "A", TTL: 10, Target: "203.0.113.42"}}})
+	assert.ElementsMatch(records, []ovhRecord{{ID: 42, Zone: "example.org", ovhRecordFields: ovhRecordFields{SubDomain: "ovh", FieldType: "A", TTL: 10, Target: "203.0.113.42"}}, {ID: 24, Zone: "example.org", ovhRecordFields: ovhRecordFields{SubDomain: "ovh", FieldType: "NS", TTL: 10, Target: "203.0.113.42"}}})
```

Comment:
Why do we need to touch a provider specific test?

Reply:
I would love to not to touch it but it started to fail with NS record support because NS records are started to get created :) If you look at https://github.com/kubernetes-sigs/external-dns/blob/ba46a498579165e220bcbfaaba9aa91fd521e5ea/provider/ovh/ovh_test.go#L91 , NS record is there in some form of test fixture